guide/src/format/configuration/general.md

```diff
@@ -46,15 +46,19 @@ This is general information about your book.
   `src` directly under the root folder. But this is configurable with the `src`
   key in the configuration file.
 - **language:** The main language of the book, which is used as a language attribute `<html lang="en">` for example.
+- **logo:** Path to a logo to shown at the top of the navigation bar. If this
+  is given as a relative path, it's base directory is the source directory.
```

Comment:
```suggestion
  is given as a relative path, its base directory is the source directory.
```

Reply:
Which content types do we support for the logo?

Reply:
I'd have to further look into that, but it seems like pixel as well as vector graphics are supported.

Reply:
Yeah, `img` supports SVG. You can simply reference the HTML spec for `img` to state which image formats are supported.